__tests__/pet.test.js

```diff
@@ -0,0 +1,139 @@
+const Pet = require("../src/pet");
+let pet = new Pet('Fido');
```

Comment:
Consider replacing it by `let pet;` only. Inside of the beforeEach() your assigning the value already, so no need to have it twice.